### PR TITLE
[Refactor] Twig cleanup: Remove core.getCsrfToken() from twig files

### PR DESCRIPTION
**Pull request by zjxiaohan**
### Please check if the PR fulfills these requirements:

* [x] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)

### What is the current behavior?
Partially addresses #2273. Currently, we use `core.getCsrfToken()` in twig files to obtain CSRF token.

### What is the new behavior?
Views pass CSRF token to twig files so that twig does not touch `core`. The whole point of using twig is not to have php mixed anyways.

### CODE REVIEW

File: site/app/templates/admin/LateDays.twig [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <div class="content">
     <h1>Late Days Allowed</h1>
     <form id="lateDayForm" method="post" enctype="multipart/form-data" onsubmit="return updateLateDays($(this));">
-        <input type="hidden" name="csrf_token" value="{{ core.getCsrfToken() }}" />
+        <input type="hidden" name="csrf_token" value="{{ csrf_tokeeen }}" />
```
Comment (sak6lab) [inline site/app/templates/admin/LateDays.twig]:
```suggestion
        <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
```

Comment (zjxiaohan) [reply site/app/templates/admin/LateDays.twig]: Oops. Thank you so much!!

**Comment by bmcutler** [comment]
@zjxiaohan please resolve merge conflicts

**Comment by xprilion** [comment]
Suggest merging this after #3817

**Pull request merged**